doc/conf.py

```diff
@@ -15,7 +15,7 @@
 import warnings
 import re
 from datetime import datetime
-from packaging.version import parse
```

Comment:
`packaging` was installed in the environment, this is not needed since we are vendoring `packaging`